cmd/build/command.go

```diff
@@ -80,21 +83,30 @@ func Build(ctx context.Context) *cobra.Command {
 	return cmd
 }
 
-func (bc *Command) getManifestAndBuildVersion(options *types.BuildOptions) (*model.Manifest, bool) {
-	manifest, errManifest := bc.GetManifest(options.File)
-	if errManifest != nil {
-		oktetoLog.Debug("error getting manifest v2 from file %s: %v. Fallback to build v1", options.File, errManifest)
+func (bc *Command) getBuilder(options *types.BuildOptions) (Builder, error) {
+	var builder Builder
+
+	manifest, err := bc.GetManifest(options.File)
+	if err != nil {
+		var invalidManifestErr *oktetoErrors.InvalidManifestError
```

Comment:
I just uploaded a new solution, I think it is one of the cleanest we can have in our case. Since we are facing a wrapped error we cannot use the `==` operator in this case. We must use the Is() method. Also, in the Golang documentation it is recommended to compare errors using Is() instead of the `==` operator.